tests/shutdown/shutdown.pm

```diff
@@ -13,6 +13,9 @@ use power_action_utils 'power_action';
 use utils;
 
 sub run {
+    my $self = shift;
+    select_console('root-console');
+    script_run('systemctl list-timers --all');
```

Comment:
i would recommend to replace this with `$self->save_and_upload_log('systemctl list-timers --all', '/tmp/all_timers.log', {screenshot => 1});` . that will store a file in the *Logs and Assets* wdyt?

Reply:
Hi, upload won't work because `shutdown` is run after `cleanup_before_shutdown` that takes down the network. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/0385b0e789210c0ac7d6af10b9c9bf3a0c32331e/tests/shutdown/cleanup_before_shutdown.pm#L57-L62
and https://openqa.suse.de/tests/9230015#step/shutdown/6

I guess it should be enough to print it since it's a rather small output and easy to read, but open to ideas

Reply:
Seems like this breaks the test https://openqa.suse.de/tests/9239946#step/shutdown/2.
I would recommend to use https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/Utils/Systemd.pm#L58 instead of `script_run`